cms/djangoapps/contentstore/views/course.py

```diff
@@ -693,6 +693,9 @@ def create_new_course_in_store(store, user, org, number, run, fields):
     definition_data = {'wiki_slug': wiki_slug}
     fields.update(definition_data)
 
+    # Set default language from settings
+    fields.update({'language': getattr(settings, 'DEFAULT_COURSE_LANGUAGE', 'en')})
```

Comment:
Please extend the whatever unit test that covers create_new_course_in_store to make sure that it sets the language properly on creation.